pkg/scheduler/algorithmprovider/defaults/defaults.go

```diff
@@ -74,6 +77,11 @@ func ApplyFeatureGates() {
 		factory.RemovePredicateKeyFromAlgorithmProviderMap(predicates.CheckNodeDiskPressurePred)
 		factory.RemovePredicateKeyFromAlgorithmProviderMap(predicates.CheckNodePIDPressurePred)
 
+		excludedPredicates = sets.NewString(predicates.CheckNodeConditionPred,
```

Comment:
not against this logic but perhaps having a map[predicate.Name]=exclude and map[predicate.Name]=includedlist might make association much easier.

Reply:
Well the underlying type for a Set is a map, and if you want to keep the `FeatureDependency` type as it is then we need the separate lists anyway. I also think having them separate is a little clearer

Reply:
Fine with me but let's see if anyone else has any other opinions on it.